CreateSalesEmployeeTable_1.sql

```diff
@@ -0,0 +1,44 @@
+USE JCAC_CormacF
+DELIMITER $$
+DROP PROCEDURE IF EXISTS CreateSalesEmployeeTable_1 $$
+CREATE PROCEDURE CreateSalesEmployeeTable_1()
```

Comment:
Your procedure name doesn't look to be the same format as merge request 4... Could you check what convention your team want to go for?